backend/kind/kubeconfig.sh

```diff
@@ -4,6 +4,9 @@ set -ex
 . ../../include/common.sh
 . .envrc
 
-cp $(kind get kubeconfig-path --name="$CLUSTER_NAME") kubeconfig
+# Kind above version 0.6.0 generates the config itself
+if [ ! -f kubeconfig ]; then
+  cp $(kind get kubeconfig-path --name="$CLUSTER_NAME") kubeconfig
```

Comment:
We are deprecating this option, check the release notes for the alternative, it will be removed soon https://github.com/kubernetes-sigs/kind/releases/tag/v0.6.0